.openshiftio/booster.yaml

```diff
@@ -1,2 +1,6 @@
 name: Vert.x - HTTP & Config Map
 description: Simple HTTP endpoint where the Vert.x application uses OpenShift ConfigMap to get the application parameters.
+  - id: community
+    name: Eclipse Vert.x 3.4.2.Final (Community)
+  - id: redhat
+    name: Vert.x 3.4.2.redhat-4 (RHOAR)
```

Comment:
Is that the final name? Until we have the name validated by "legal", we can't use it. That would also mean that we need to change the booster and also the catalog after each ER build

Reply:
In a conversation with John, this is the naming he wanted to use.  The thought was that in the future, we may want to support multiple versions of either product or community, thus including the specific versions in the naming.

For the catalog, I believe this will be a problem independent of the labels in the .openshiftio/booster.yaml files.  We're going to need something like a "released" tag and a "next" tag to indicate what should be used by the actual Launch application (released) and what's currently being worked on (next).  We'll need to make a conscious effort to move the tags appropriately.